### DISCUSSION TITLE
Problem when updating Jumper T-Pro V2 with buddy

### DISCUSSION THREAD

**Discussion by RoronoaVector** [EdgeTX/edgetx#4137] [Q&A] [answered]
Hi,

Im facing a problem when i try to use the edgeTx buddy to update my firmware, when i try to flash it throught an usb i get this error "Failed to execute 'open' on 'USBDevice': Access denied." Im trying to do it like this beacuse if i install the firmware with my jumper it seems like it have a bug or something beacuse the lcd its mirrored and seems that it doesnt work properly. Any ideas about this?

If i put manualñly the latest version my jumper T Pro is not able to start, im trying to put a 2.8.x version, this version works but with the screen mirrored and without any map of the controls

Thx in advance

**Comment by pfeerick** [comment, marked as answer]
If the display is mirrored, you are flashing the wrong firmware. Are you
flashing the TproV2 firmware after enabling the prereleases filter so you
can flash nightly firmware. You should *not* be flashing the TPro firmware.

What OS are you on? That error from Buddy could be driver or permissions
related.

**Reply by RoronoaVector** [reply]
Im on windows but i tried in MacOS aswell, in windows i can select the STM32 bootloader but it seems that when Buddy tried to flash the firmware he dont have enought permissions to acces, anyways i could flashed manually with the radio downloading the "nightly" version you told me.

Thx a lot for the help.
